Documentation/ApiOverview/Hooks/EventDispatcher/Index.rst

```diff
@@ -149,10 +148,12 @@ The custom PHP class :php:`MyCompany\MyPackage\EventListener\NullMailer` serves
 whereas the :yaml:`identifier` is a common name so orderings can be built upon the identifier,
 the optional :yaml:`before` and :yaml:`after` attributes allow for custom sorting against :yaml:`identifier`.
 
-The :yaml:`event` attribute is the Fully Qualified Name of the Event object.
-
 If no attribute :yaml:`method` is given, the class is treated as Invokable, thus :php:`__invoke` method is called.
 
+.. versionchanged:: 11.3
```

Comment:
I would explain this a bit further: The event tag can be omitted IF the listener implementation has a corresponding event type in the method signature. If the listener - for whatever reason - listens to multiple events or does not have the type specified this will fail and the event has to be stated explicitly again.